ui/app/components/clients/current.js

```diff
@@ -7,9 +7,6 @@ export default class Current extends Component {
     { key: 'entity_clients', label: 'entity clients' },
     { key: 'non_entity_clients', label: 'non-entity clients' },
   ];
-  @tracked firstUpgradeVersion = this.args.model.versionHistory[0].id || null; // return 1.9.0 or earliest upgrade post 1.9.0
```

Comment:
turned into a getter because this should only ever be set/updated by the model passed to the component